README.md

```diff
@@ -573,8 +615,8 @@ function* test() {
 * Return address not found in frame
 * Should not directly enter OSR-compiled function
 * Sloppy function expects JSReceiver as receiver.
-* Smi addition overflow
-* Smi subtraction overflow
+* ~~Smi addition overflow~~
```

Comment:
I am not sure) See: https://github.com/vhf/v8-bailout-reasons/issues/16 and https://github.com/vhf/v8-bailout-reasons/commit/939e00e054fe2af31b3f8abfac1edd5a3c0826f1